docs/source/deployguide/cadeploy.md

```diff
@@ -0,0 +1,597 @@
+
+```
+./fabric-ca-client enroll -d -u https://rcaadmin:rcaadminpw@my-machine.example.com:7054 --tls.certfiles ../tls/tls-ca-cert.pem -M msp-rca --csr.hosts 'host1,*.example.com'
+```
+
+The organization root CA TLS signed certificate is generated under `fabric-ca-client/tls-ca/admin/msp-rca/signcert` and the private key is available under `fabric-ca-client/tls-ca/admin/msp-rca/keystore`. When you deploy the organization CA you will need to point to the location of these two files in the `tls` section of the CA configuration .yaml file.
+
+### (Optional) Register and enroll the Intermediate CA admin with the TLS CA
+
+Similarly, if you are planning to have an intermediate CA in addition to a root CA, you should also register and enroll that intermediate CA admin user now as well. The following command registers the intermediate CA admin id `icaadmin` and `icaadminpw` with the TLS CA. You can use any values you choose for the identity name and password.
+
+```
+./fabric-ca-client register -d --id.name icaadmin --id.secret icaadminpw -u https://my-machine.example.com:7054  --tls.certfiles ../tls/tls-ca-cert.pem
+```
+
+Now would also be a good time to generate the intermediate CA TLS certificates for the `icaadmin` user by enrolling the user. Again, in this case, we use the `-M` flag on the enroll command to designate where the generated intermediate CA TLS certificates should be stored for the `icaadmin` user. Using the `-M` flag we will put them in a new folder named `msp-ica`.
+
+```
+./fabric-ca-client enroll -d -u https://icaadmin:icaadminpw@my-machine.example.com:7054 --tls.certfiles ../tls/tls-ca-cert.pem -M msp-ica --csr.hosts 'host1,*.example.com'
+```
+
+The intermediate CA TLS signed certificate is generated under `fabric-ca-client/tls-ca/admin/msp-ica/signcert` and the private key is available under `fabric-ca-client/tls-ca/admin/msp-ica/keystore`. When you deploy the intermediate CA you will need to refer to these two files in the `tls` section of the intermediate CA configuration .yaml file.
+
+**Tip:** After you have registered all your nodes with the TLS CA, it can be safely turned off.
+
+## Deploy an organization CA
+
+The deployment process overview describes the need for both an enrollment CA and a TLS CA for every organization. The TLS CA issues the TLS certificates that allow for secure transactions within the organization. The enrollment CA, also referred to as the "organization CA" or the "eCert CA" is used to issue identities for the organization. You deployed the TLS CA in the previous set of steps, now we are ready to deploy the organization CA. Later in this topic we will also create an intermediate CA; therefore, this CA serves as the "root CA" in that chain of trust.
```

Comment:
Switching between calling it "enrollment CA" and "organization CA" has been confusing. Can we just pick one term and use it consistently throughout?

Reply:
Thank you. I'm going with organization CA.